### PR TITLE
🚨 Stop checking flow_server_enabled and remove support for editing it

### PR DESCRIPTION
**Pull request by rowanseymour**
Don't want to ever get into a state where this is false

### CODE REVIEW

File: temba/msgs/models.py [hunk 1]
```diff
@@ -957,8 +957,8 @@ def handle(self):
 
         if settings.TESTING:
             legacy.handle_message(self)
-
-        mailroom.queue_msg_handling(self)
+        else:  # pragma: no cover
```
Comment (rowanseymour) [inline temba/msgs/models.py]: fixes seeing mailroom errors during tests

**Comment by norkans7** [comment]
I think we need to remove https://github.com/nyaruka/rapidpro/blob/master/temba/channels/types/twilio/type.py#L41 too

**Comment by nicpottier** [comment]
Norbert brings up an interesting point there. We need the conversion TO flow server enabled to exist even if flow server enabled doesn't exist. We could still have a migration that does this for v5 that we then remove for v6..

**Comment by rowanseymour** [comment]
@nicpottier @norkans7 exactly - I'm the first person to complain about tying migrations to actual code - but I previously added a call to `Org.enable_flow_server` to https://github.com/nyaruka/rapidpro/blob/master/temba/orgs/migrations/0053_migrate_all_orgs.py with the caveat that we would be squashing migrations before removing any of that code. So the `enable_for_server` methods do have to remain for now (this PR just moves that method into the migration), and after the migration has been squashed, we can clean up the code.

**Pull request merged**